tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -358,6 +358,13 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string PropagationStyle = "DD_TRACE_PROPAGATION_STYLE";
 
+        /// <summary>
+        /// Configuration key to configure if propagation should only extract the first header once a configure
+        /// propagator extracts a valid trace context.
+        /// </summary>
+        /// <seealso cref="TracerSettings.PropagationExtractFirst"/>
+        public const string PropagationExtractFirst = "DD_TRACE_PROPAGATION_EXTRACT_FIRST";
```

Comment:
I'm sure it's too late to change this, but I think adding the word "only" to this name would make it much more clear, like `DD_TRACE_PROPAGATION_ONLY_EXTRACT_FIRST` or `DD_TRACE_PROPAGATION_EXTRACT_FIRST_ONLY`. 🤷🏽‍♂️

Reply:
Going against doing the change for now since other languages already released with the recommended setting.